### ISSUE TITLE
Jira - User is presented with a warning about app being in development mode during the oauth workflow for Jira

### ISSUE THREAD

**Issue by sangee2004** [obot-platform/obot#1015]
**Steps to reproduce the problem:**
1. Configure Atlassian Provider.
2. Create an agent with `Jira` tool bundle.
3. Chat with the agent and ask about Jira 
4. When user is presented with the Jira oauth workflow , noice that following waring being presented to the user
<img width="737" alt="Screenshot 2024-12-20 at 11 53 23 AM" src="https://github.com/user-attachments/assets/f1df5fe4-4e58-48a7-b769-2fac7a6821f7" />

**Expected Behavior:**
Ideally we should not be presented with this warning.

Comments from @njhale  - Yeah, you have to go through one more step to make it a production app in the atlassian dev console

**Comment by tybalex** [comment]
This warning is not presented by us, but Atlassian-jira. 
To remove it, the owner of the Atlassian app can go to the developer console and change the distribution  status of the app to `sharing`, and then the warning will disappear.

the distribution status:
<img width="911" alt="Screenshot 2025-01-02 at 2 28 33 PM" src="https://github.com/user-attachments/assets/82bad5c4-fd94-491e-acb5-03c620354365" />

the warning is gone:
<img width="911" alt="Screenshot 2025-01-02 at 2 28 33 PM" src="https://github.com/user-attachments/assets/ef57ee26-83f8-45cf-b99f-bbcb69fedb3f" />

So, basically there is nothing we need to fix on our end.
can you confirm?

**Comment by sangee2004** [comment]
@tybalex With the steps done in https://github.com/obot-platform/obot/issues/1015#issuecomment-2567339886 , I dont see the warnings relating to "Make sure to trust Obot" anymore during the oath workflow.


Reopening this issue to include these steps as part of `Configure Atlassian OAuth App`steps when configuring `Atlassian` app.

**Comment by tybalex** [comment]
@sangee2004 included theses steps in the merged PR.

**Comment by sangee2004** [comment]
Tested with latest build. Following information is added to steps in `Configure Atlassian OAuth App`

<img width="770" alt="Image" src="https://github.com/user-attachments/assets/70026c9b-321a-48df-b850-ccf591cc819c" />
